autonomy/data/Dockerfiles/agent/Dockerfile

```diff
@@ -1,34 +1,19 @@
-ARG AEA_VERSION=latest
-FROM valory/open-aea-user:${AEA_VERSION}
-ARG AUTHOR=default_author
-ARG AEA_AGENT
+ARG AUTONOMY_IMAGE_VERSION="latest"
+ARG AUTONOMY_IMAGE_NAME="valory/open-autonomy"
 
-COPY openssl.cnf /etc/ssl
-COPY Pipfile .
-COPY . /home/ubuntu
+FROM ${AUTONOMY_IMAGE_NAME}:${AUTONOMY_IMAGE_VERSION}
 
-WORKDIR /home/ubuntu
-ENV PATH=$PATH:/home/ubuntu/.local/bin
+ARG AEA_AGENT
```

Comment:
I think @8ball030 was once telling me all args need to be listed up top?

Reply:
Noted on the right line